src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -132,5 +134,11 @@ object Instance {
   implicit val idFormat = Json.format[Instance.Id]
   implicit val instanceStatusFormat = Json.format[InstanceStatus]
   implicit val instanceStateFormat = Json.format[InstanceState]
-  implicit val instanceJsonFormat = Json.format[Instance]
+  implicit val instanceJsonFormat: Format[Instance] = Json.format[Instance]
+
+  import mesosphere.mesos.Placed
+  implicit class PlacedInstance(instance: Instance) extends Placed {
```

Comment:
As far as I can read, this just adds the marker trait, so if I follow the logic (whether it's view/context or implicit), if I have the implicit placement in my list, it's automatically now tagged? This seems wrong to me.